card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+      },
+      {
+        "name": "simple skin",
+        "type": "Skin",
+        "content": "[[style: jquery-ui-smoothness]]\n[[style: functional]]\n[[style: standard]]\n[[style: right sidebar]]\n[[style: common]]\n[[style: traditional]]\n[[style: glyphicons]]\n[[style: bootstrap compatible]]"
+      },
```

Comment:
fwiw, I was kind of reworking "style: traditional" to be, effectively "bootstrap compatible".  This might be a better approach, though...